libs/shared/ui/src/lib/components/help-section/help-section.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { render } from '@testing-library/react'
+
+import HelpSection from './help-section'
+
+describe('HelpSection', () => {
+  it('should render successfully', () => {
+    const description = 'Test description'
+    const { baseElement } = render(<HelpSection description="Test description" />)
```

Comment:
You can declare your props outside and use a beforeEach to add your props for each test. (You have exemple in other components)